hrui/js/src/hrui.js

```diff
@@ -160,6 +175,15 @@
   }
 
   /**
+   * Checks if the quick search field itself has any current value
+   *
+   * @return {boolean}
+   */
+  function isQuickSearchOnGoing () {
+    return $('#sort_name_navigation').val().length;
```

Comment:
what if `$('#sort_name_navigation').val() === '        '` don't we need to trim()?

this is not a change request, up to you.

Reply:
changed it to
```js
return !!$('#sort_name_navigation').val().trim();
```